### DISCUSSION TITLE
Did not receive my chia to the chia wallet

### DISCUSSION THREAD

**Discussion by Vish513** [Chia-Network/chia-blockchain#18241] [Ideas]
Hello,

A few days ago I sent 14 chia to my wallet
I checked the address several times, it was correct, but the chia did not arrive on it.

**Comment by Vish513** [comment]
I installed the new version of chia wallet and everything became normal.

**Comment by BrandtH22** [comment]
Hey @Vish513 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

The first thing i would verify is that the wallet is fully synced (is the wallet icon in the top right green?). There are some reported syncing issues with the 2.4.0 version, I recommend installing the 2.4.1 rc version from here and letting it sync fully:
https://github.com/Chia-Network/chia-blockchain/releases/tag/2.4.1-rc1
